### ISSUE TITLE
Group Home > Privacy status text is incorrect

### ISSUE THREAD

**Issue by bree-z** [cuny-academic-commons/commons-in-a-box#490]
The text that appears on the Group Home stating the privacy settings of the group/site ("This [Group] is Open...") is incorrect in the following cases below. The text on CBOX OpenLab seems to be different from the City Tech OpenLab (checked also on BMCC and [cboxopenlab.org](https://cboxopenlab.org/)) and I couldn't find the original ticket for this, so I tried to piece together what the correct text should be. The text that I see on https://unwavering-vole-0e3fea.instawp.xyz/ is also different from BMCC and [cboxopenlab.org](https://cboxopenlab.org/). 

**Group: Public + Site: "I would like the site to be visible to community members with a role on the associated site."**

Current text: This [Group] is OPEN, but the corresponding Site is HIDDEN.
Correct text: This [Group] is OPEN, but the corresponding Site is PRIVATE.

**Group: Private + Site: all settings except "I would like my site to be visible only to those members with an administrator role on the associated site."**

In these cases, the second part of the text, about the site settings, is correct, but the first part is not:

Current text: This Course is HIDDEN...
Correct text: This Course is PRIVATE...

**Private group + Admins-only site**

Current: This Course is HIDDEN, and you must be a member to view the corresponding Site.
Correct: This Course is PRIVATE, and the corresponding Site is HIDDEN.

**Hidden group + Admins-only site**

Current: This Course is HIDDEN, and you must be a member to view the corresponding Site.
Correct: This Course is HIDDEN, and the corresponding Site is HIDDEN.

<img width="900" alt="PrivacyStatusText" src="https://github.com/user-attachments/assets/528bbacd-2b08-434a-be83-91e3da2488cf">

**Comment by boonebgorges** [comment]
We made changes to this on the OpenLab in tickets: http://redmine.citytech.cuny.edu/issues/3074, http://redmine.citytech.cuny.edu/issues/2921

Porting these changes to CBOX-OL was not included in our initial discussions for this development cycle. I have a strong desire not to do work on CBOX-OL that will further diverge it from the City Tech OpenLab. But it's not possible to simply copy over from the OpenLab, because that depends on the functionality in http://redmine.citytech.cuny.edu/issues/2921. So I think that any changes to the text described in this ticket will have to wait until that feature is ported in a future release.

**Comment by boonebgorges** [comment]
Yes, it appears to be a result of #438. It's possible for me to roll back that change, but the change is only relevant for new installations. That means that anyone who installed CBOX-OL after version 1.5.0 but before version 1.6.0 will continue to see the old strings. This is not likely to be a large number of sites.

Before rolling back, I would encourage you to look back at #438 and http://redmine.citytech.cuny.edu/issues/3074 so that you can consider the original motivations for making the change. I don't fully understand why the team thought it was a good idea to use the 'HIDDEN' language in the first place, but presumably there was a conversation around it, and it's possible that we'll be reintroducing those motivating issues by reintroducing the 'PRIVATE' language.

**Comment by bree-z** [comment]
Thanks Boone! We'd like to go ahead and roll it back. 

We'd prefer to reintroduce the 'PRIVATE' language.

**Comment by boonebgorges** [comment]
I've made the change.

**Comment by bree-z** [comment]
Thanks Boone! How do I test this? 

I set up a fresh installation, but I was getting errors using your deployment script. And I wonder if I shouldn't have even installed CBOX OpenLab.

**Comment by boonebgorges** [comment]
The deployment script assumes that you have it set up already. You'll need to do git checkouts of the items described in that script. You can probably get away with just:

```
git clone https://github.com/cuny-academic-commons/commons-in-a-box wp-content/plugins/commons-in-a-box
git clone https://github.com/cuny-academic-commons/commons-in-a-box wp-content/plugins/cbox-openlab-core
git clone https://github.com/cuny-academic-commons/commons-in-a-box wp-content/themes/openlab-theme
```

and then running through the rest of the setup process in the Dashboard.

**Comment by bree-z** [comment]
Thanks Boone! I did that and I'm getting the following errors:

Deploying cbox-openlab-core 1.6.x...
error: pathspec '1.6.x' did not match any file(s) known to git
fatal: 'origin' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Deploying openlab-theme 1.6.x...
error: pathspec '1.6.x' did not match any file(s) known to git
fatal: 'origin' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Deploying commons-in-a-box 1.6.x...
error: pathspec '1.6.x' did not match any file(s) known to git
fatal: 'origin' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

etc...

**Comment by boonebgorges** [comment]
I don't know that you're going to be able to run this script on another server without my ssh keys. The instructions I gave you are to get the clones in place, and then you'll have to do the checkouts yourself.

If this is too difficult, I recommend (a) not bothering to test this (it's a minor text change) or (b) wiping the database on your existing test rig and reinstalling

**Comment by bree-z** [comment]
Thanks Boone. I'm inclined to go with (a) and not bother testing this. 

I'll check in with Charlie first though. I can't seem to find her username to tag her here.

**Comment by bree-z** [comment]
We can go ahead and close this.
